Client/Frontend/Strings.swift

```diff
@@ -5356,6 +5376,11 @@ extension String {
         tableName: nil,
         value: nil,
         comment: "Accessibility label for the tabs button in the (top) tab toolbar")
+    public static let TabsButtonShowTabsLargeContentTitle = MZLocalizedString(
```

Comment:
I think you can put this string with the other ones above, using the same `tableName`. The string ordering in this file is a bit out of order when it's older strings.